Makefile

```diff
@@ -20,15 +20,31 @@ deploy: undeploy
 	oc create -f deploy/role.yaml
 	oc create -f deploy/role_binding.yaml
 	oc create -f deploy/crds/rhjmc_v1alpha1_flightrecorder_crd.yaml
+	oc create -f deploy/crds/rhjmc_v1alpha1_containerjfr_crd.yaml
+	oc create -f deploy/containerjfr_command_config_map.yaml
+	oc create -f deploy/containerjfr_config_map.yaml
 	sed -e 's|REPLACE_IMAGE|$(IMAGE_TAG)|g' deploy/operator.yaml | oc create -f -
+	oc create -f deploy/crds/rhjmc_v1alpha1_containerjfr_cr.yaml
+	oc create -f deploy/exposecontroller_config_map.yaml
+	oc create -f deploy/exposecontroller.yaml
 
 .PHONY: undeploy
 undeploy:
+	- oc delete deployment exposecontroller
+	- oc delete configmap exposecontroller
+	- oc delete all -l project=exposecontroller
+	- oc delete routes -l generator=exposecontroller
 	- oc delete deployment container-jfr-operator
 	- oc delete containerjfr --all
 	- oc delete flightrecorder --all
 	- oc delete all -l name=container-jfr-operator
+	- oc delete all -l app=containerjfr
+	- oc delete all -l app=containerjfr-pod
```

Comment:
I can't seem to find where this label is used, I think the pod is also under the `app=containerjfr` label.

Reply:
Good call, probably a holdover from some in-between iteration.